### ISSUE TITLE
includeObject for 'getCurrentUserFromServer' and user signup/signin

### ISSUE THREAD

**Issue by JTPenn-EG** [parse-community/Parse-SDK-Flutter#767]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest version of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse Flutter SDK](https://github.com/parse-community/Parse-SDK-Flutter/releases). 

### Issue Description

No option to includeObjects for User function 'getCurrentUserFromServer'

### Steps to reproduce

Just add a pointer to your User class and the data returned from 'getCurrentUserFromServer' contains the pointer info only (object id of the pointer only)

### Actual Outcome

..

### Expected Outcome

Expected to be able to use includeObject in 'getCurrentUserFromServer' or in user sign up or user sign in functions to retrieve extra info of user pointer fields

### Environment

Parse Flutter SDK
- SDK version: `3.1.2`
- Operating system version: `Android`

Server
- Parse Server version: `-`

**Comment by JTPenn-EG** [comment]
Unfortunately, I just realized this is not yet supported by Parse Server
[#8095](https://github.com/parse-community/parse-server/issues/8095)

**Comment by mbfakourii** [comment]
We have such a function in [parse User](https://github.com/parse-community/Parse-SDK-Flutter/blob/60ca5a4d47dd98f86e130e6455731dff216ebed8/packages/dart/lib/src/objects/parse_user.dart#L102)

Can you explain more in code format.

**Comment by mbfakourii** [comment]
Yes, unfortunately, it is not currently supported, the only request is again with the query

**Comment by mbfakourii** [comment]
Yes, I also tested it, it seems to have a problem.
The only way is to use a query

**Comment by mbfakourii** [comment]
The function `getCurrentUserFromServer` uses the address `https://parseapi.back4app.com/users/me`

@JTPenn-EG 
I tried to solve your problem with `https://parseapi.back4app.com/users/me?include=myClassName`, but it didn't work from Parse's server side! And it did not include the class!

@mtrezza 
It seems to be a limitation on the Parse server side!

**Comment by mtrezza** [comment]
Could you give an example (for the Parse Server REST API or Parse JS SDK) of:
- request
- expected response
- actual response
